minio/minio
# fix: heal optimizations in crawler to avoid multiple healing attempts

## Description
fix: heal optimizations in crawler to avoid multiple healing attempts

## Motivation and Context
Fixes two problems

- Double healing when bitrot is enabled, instead heal attempt
  once in applyActions() before lifecycle is applied.

- If applyActions() is successful and getSize() returns proper
  value, then object is accounted for and should be removed
  from the oldCache namespace map to avoid double heal attempts.

## How to test this PR?
delete 10,000 files see the usage build up and then delete 
them to see how quickly we re-update the usage count, this can be
run with `warp get --duration 10m` 

## Types of changes

## Checklist:
